puppet/zulip/manifests/postgres_common.pp

```diff
@@ -46,6 +46,18 @@
       exec {'pip3_deps':
         command => 'python3 -m pip install pytz python-dateutil'
       }
+      file { '/usr/bin/pg_config':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/pg_config"
+      }
+      file { '/usr/bin/pg_isready':
+        ensure => 'link',
+        target => "/usr/pgsql-${zulip::base::postgres_version}/bin/pg_isready"
+      }
```

Comment:
I wonder if we should replace our one use of `pg_isready` with like `psql -c ''` to avoid the need to pollute `/usr/bin` with symlinks.

Reply:
We only use it ourselves in a couple places, so that'd almost certainly be a cleaner solution than creating these symlinks.  

Though actually, the places I see us using it are both for the development environment, not production, so why would we need this puppet change?

Reply:
> Though actually, the places I see us using it are both for the development environment, not production, so why would we need this puppet change?

I assumed that `pg_config`, `pg_isready`, and `psql` are also required in production environment. Now that it is clear that only `pg_config` is used in production, I will remove the latter 2.

Reply:
I found that `psql` is used at puppet/zulip/files/postgresql/pg_backup_and_purge and several other places. So I will remove only `pg_isready`.